### ISSUE TITLE
disable http certificate verification

### ISSUE THREAD

**Issue by Gaurang033** [peritus/robotframework-httplibrary#36]
Hi, 

Would you please let me know if there is any way to disable to http certification verification. I search on internet and figure out there is such thing for httplib however nothing so far for httplib and I guess you use HttpLib

If there is any way to implement it let me konw as well, as I stuck at this point where I can't use you lib until I resolve the issue of certification. 

my website uses the invliad certification and until I add that to trusted store using IE there is not way I can use your API, all API are failing with HTTPS certification error. 

I tried to find out, way to install certificate however it is not possible until I have admin rights, which I don't. 

would appreciate your quick response.

**Comment by jvanz** [comment]
Yeah, I need this configuration too. I believe it will be easy to add it to the lib. We need change this [line](https://github.com/peritus/robotframework-httplibrary/blob/master/src/HttpLibrary/livetest.py#L104) and add a extra argument:

`self.conn[scheme] = conn_classes[scheme](self.host, context=ssl._create_unverified_context())`
